release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -2567,7 +2566,7 @@ link:https://access.redhat.com/solutions/6762051[{product-title} 4.9.23 containe
 [id="ocp-4-9-23-known-issues"]
 ==== Known Issues
 
-{product-title} release 4.9.23 references images whose manifests use the `application/vnd.oci.image.manifest.v1+json` media type. This might cause problems when mirroring to image registries that do not support OCI media types. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2059762[*BZ#2059762*])
+{product-title} release 4.9.23 references images whose manifests use the `application/vnd.oci.image.manifest.v1+json` media type. This might cause problems when mirroring to image registries that do not support OCI media types. There is currently no work around for this issue and it planned to be fix in a future version of {product-title} 4.9. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2059762[*BZ#2059762*])
```

Comment:
```suggestion
* {product-title} release 4.9.23 references images whose manifests use the `application/vnd.oci.image.manifest.v1+json` media type. This might cause problems when mirroring to image registries that do not support OCI media types. There is currently no work around for this issue and it planned to be fixed in a future version of {product-title} 4.9. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2059762[*BZ#2059762*])
```